lib/graphql/language/parser.rb

```diff
@@ -1,7 +1,7 @@
 #
 # DO NOT MODIFY!!!!
-# This file is automatically generated by Racc 1.4.15
-# from Racc grammer file "".
+# This file is automatically generated by Racc 1.4.16
```

Comment:
I used the rake task, but I patched the Rakefile to bypass the version check because I had trouble installing the required version. `2.7.1` comes bundled with `1.4.16`, so yes it would make sense to update the `Rakefile`. I'll update my PR.

Reply:
Done.